application/classes/Ushahidi/Repository/Form/Stats.php

```diff
@@ -335,8 +337,8 @@ public function getPostCountByDataSource($form_id, $created_after, $created_befo
             'email' => $dataSourceCounts['email'],
             'twitter' => $dataSourceCounts['twitter'],
             'web' => $this->queryForWeb($form_id, $created_after, $created_before),
-            'all' => $this->queryForAllPosts($form_id, $created_after, $created_before)
         ];
+        $result['all'] = $result['web'] + $result['email'] + $result['twitter'] + $result['sms'];
```

Comment:
if we just send a queryForAllPosts, we end up with an incorrect number of posts due to translation posts (should be type=report for normal posts)